crates/ra_syntax/src/ast/make.rs

```diff
@@ -75,6 +75,14 @@ pub fn record_field(name: ast::NameRef, expr: Option<ast::Expr>) -> ast::RecordF
     }
 }
 
+pub fn record_field_def(name: ast::NameRef, ty: ast::TypeRef) -> ast::RecordFieldDef {
+    return from_text(&format!("{}: {}", name, ty));
+
+    fn from_text(text: &str) -> ast::RecordFieldDef {
```

Comment:
why the nested function?

Reply:
100% copypasta from other functions in `ast::make`